parsl/executors/serialize/codeutil.py

```diff
@@ -23,12 +23,14 @@ def code_ctor(*args):
 
 
 def reduce_code(co):
-    args = [co.co_argcount, co.co_nlocals, co.co_stacksize,
+    args =  [co.co_argcount, co.co_nlocals, co.co_stacksize,
```

Comment:
Is this extra whitespace intentional ?

Reply:
It comes from re-copying the original at ipyparallel/serialize/codeutil.py where it is indented that way. i've changed it back.